recipes/jemalloc/all/test_package/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 cmake_minimum_required(VERSION 3.1)
-project(test_package)
+project(test_package LANGUAGES CXX)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
-add_executable(${PROJECT_NAME} test_package.c)
+add_executable(${PROJECT_NAME} test_package.cpp)
```

Comment:
Isn't this change a symptom that we need to add stdcpp library as a system library?
Changing the test from c++ to c is hiding the problem, IMHO.

Reply:
The test is written only to check the usage of c facilities of jemalloc library. In this library there is an option to support some c++ features. Given this, when c++ is enabled this test will fail, just because it is supposed to be used only in c case. I'm not an author of this test and just made it work as it should, I wonder how it even passed the ci when it was submitted at the first time. In my opinion, linking stdc++ library to c binary makes no sense at all. I hope i stated my opinion in the way you can understand and if you still thinking that this is hiding the problem, I would need to write additional test only for c++ case.